src/diffusers/models/unet_2d_condition.py

```diff
@@ -105,15 +105,16 @@ def __init__(
             "CrossAttnDownBlock2D",
             "DownBlock2D",
         ),
-        mid_block_type: str = "UNetMidBlock2DCrossAttn",
+        mid_block_type: Optional[str] = "UNetMidBlock2DCrossAttn",
         up_block_types: Tuple[str] = ("UpBlock2D", "CrossAttnUpBlock2D", "CrossAttnUpBlock2D", "CrossAttnUpBlock2D"),
-        only_cross_attention: Union[bool, Tuple[bool]] = False,
+        only_cross_attention: Union[bool, Tuple[bool], None] = False,
         block_out_channels: Tuple[int] = (320, 640, 1280, 1280),
         layers_per_block: int = 2,
         downsample_padding: int = 1,
         mid_block_scale_factor: float = 1,
         act_fn: str = "silu",
-        norm_num_groups: int = 32,
+        norm_num_groups: Optional[int] = 32,
+        norm_group_size: Optional[int] = 32,
```

Comment:
What is the group size for the upscaling model here?

Reply:
32